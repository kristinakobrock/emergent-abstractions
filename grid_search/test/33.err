  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:02, 22.05it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 23.80it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 22.81it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 22.51it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 24.40it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 21.62it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 21.80it/s] 48%|############################################1                                               | 24/50 [00:01<00:01, 22.89it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 26.13it/s] 64%|##########################################################8                                 | 32/50 [00:01<00:00, 27.30it/s] 70%|################################################################3                           | 35/50 [00:01<00:00, 24.54it/s] 76%|#####################################################################9                      | 38/50 [00:01<00:00, 25.79it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 28.07it/s] 90%|##################################################################################8         | 45/50 [00:01<00:00, 27.33it/s] 96%|########################################################################################3   | 48/50 [00:01<00:00, 28.03it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.62it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 15%|##############3                                                                              | 2/13 [00:00<00:00, 18.81it/s] 38%|###################################7                                                         | 5/13 [00:00<00:00, 23.22it/s] 62%|#########################################################2                                   | 8/13 [00:00<00:00, 22.55it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 25.04it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.68it/s]
