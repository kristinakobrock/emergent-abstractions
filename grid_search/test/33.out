# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6968682408332825, "acc": 0.5866674184799194, "length": 3.516935348510742, "mode": "train", "epoch": 1}
{"loss": 0.6812800168991089, "acc": 0.5707030892372131, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6706384420394897, "acc": 0.6112409234046936, "length": 3.465794324874878, "mode": "train", "epoch": 2}
{"loss": 0.6707659959793091, "acc": 0.610546886920929, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6632515788078308, "acc": 0.6237513422966003, "length": 3.4488637447357178, "mode": "train", "epoch": 3}
{"loss": 0.6724931001663208, "acc": 0.6095702648162842, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6602937579154968, "acc": 0.6274899244308472, "length": 3.4621803760528564, "mode": "train", "epoch": 4}
{"loss": 0.6700423955917358, "acc": 0.6187499761581421, "length": 1.12890625, "mode": "test", "epoch": 4}
{"loss": 0.6595361828804016, "acc": 0.6278314590454102, "length": 3.461923599243164, "mode": "train", "epoch": 5}
{"loss": 0.6609066128730774, "acc": 0.6324249505996704, "length": 1.765625, "mode": "test", "epoch": 5}
{"loss": 0.6566282510757446, "acc": 0.6279589533805847, "length": 3.4421653747558594, "mode": "train", "epoch": 6}
{"loss": 0.6508787274360657, "acc": 0.6323333382606506, "length": 3.125, "mode": "test", "epoch": 6}
{"loss": 0.6466132402420044, "acc": 0.6305569410324097, "length": 3.529451370239258, "mode": "train", "epoch": 7}
{"loss": 0.6349439024925232, "acc": 0.6480469703674316, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.6300983428955078, "acc": 0.6499415636062622, "length": 3.718660593032837, "mode": "train", "epoch": 8}
{"loss": 0.6156882643699646, "acc": 0.6701173186302185, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.6156858205795288, "acc": 0.6637179255485535, "length": 3.8315861225128174, "mode": "train", "epoch": 9}
{"loss": 0.6124392747879028, "acc": 0.6673827767372131, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.6071749329566956, "acc": 0.6696550250053406, "length": 3.87188458442688, "mode": "train", "epoch": 10}
{"loss": 0.6124355792999268, "acc": 0.6722655892372131, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.5997730493545532, "acc": 0.678061306476593, "length": 3.8884687423706055, "mode": "train", "epoch": 11}
{"loss": 0.6121314764022827, "acc": 0.6755859851837158, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.588497519493103, "acc": 0.6879544854164124, "length": 3.907797336578369, "mode": "train", "epoch": 12}
{"loss": 0.5930174589157104, "acc": 0.6927734017372131, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.5759914517402649, "acc": 0.7038187980651855, "length": 3.931743860244751, "mode": "train", "epoch": 13}
{"loss": 0.5806273221969604, "acc": 0.7101563811302185, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.5614839792251587, "acc": 0.7140501737594604, "length": 3.955132246017456, "mode": "train", "epoch": 14}
{"loss": 0.5656589269638062, "acc": 0.7197265625, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.5423415303230286, "acc": 0.7234030365943909, "length": 3.965350866317749, "mode": "train", "epoch": 15}
{"loss": 0.5611490607261658, "acc": 0.7232422232627869, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.5357514023780823, "acc": 0.7315307259559631, "length": 3.973170518875122, "mode": "train", "epoch": 16}
{"loss": 0.5616673827171326, "acc": 0.7251953482627869, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.5217797756195068, "acc": 0.7368913292884827, "length": 3.978667736053467, "mode": "train", "epoch": 17}
{"loss": 0.5494436621665955, "acc": 0.7392578721046448, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.5170305967330933, "acc": 0.7450730204582214, "length": 3.979292392730713, "mode": "train", "epoch": 18}
{"loss": 0.5526893138885498, "acc": 0.7406249642372131, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.5106624960899353, "acc": 0.7509565353393555, "length": 3.9806747436523438, "mode": "train", "epoch": 19}
{"loss": 0.5497459769248962, "acc": 0.73828125, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.5003907680511475, "acc": 0.7567053437232971, "length": 3.9834282398223877, "mode": "train", "epoch": 20}
{"loss": 0.5414454340934753, "acc": 0.7337890267372131, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.4974203109741211, "acc": 0.7594180107116699, "length": 3.9845712184906006, "mode": "train", "epoch": 21}
{"loss": 0.5391149520874023, "acc": 0.7417969107627869, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.4932447373867035, "acc": 0.7620190978050232, "length": 3.984618902206421, "mode": "train", "epoch": 22}
{"loss": 0.538835346698761, "acc": 0.7447265982627869, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.4919249415397644, "acc": 0.7620608806610107, "length": 3.9891610145568848, "mode": "train", "epoch": 23}
{"loss": 0.5326293706893921, "acc": 0.7484374642372131, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.48470455408096313, "acc": 0.7646719217300415, "length": 3.9901719093322754, "mode": "train", "epoch": 24}
{"loss": 0.5300748348236084, "acc": 0.7517578601837158, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4872550368309021, "acc": 0.7672622203826904, "length": 3.9907710552215576, "mode": "train", "epoch": 25}
{"loss": 0.5330600142478943, "acc": 0.7521483898162842, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.4795066714286804, "acc": 0.7659911513328552, "length": 3.99237060546875, "mode": "train", "epoch": 26}
{"loss": 0.5263537764549255, "acc": 0.7449217438697815, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.4796381890773773, "acc": 0.7671070098876953, "length": 3.992227077484131, "mode": "train", "epoch": 27}
{"loss": 0.5279198884963989, "acc": 0.7396484017372131, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.4733864665031433, "acc": 0.7715364098548889, "length": 3.99257230758667, "mode": "train", "epoch": 28}
{"loss": 0.5229977369308472, "acc": 0.7517578601837158, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.4658185839653015, "acc": 0.7712565064430237, "length": 3.9941153526306152, "mode": "train", "epoch": 29}
{"loss": 0.5137831568717957, "acc": 0.7548828125, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.4575583338737488, "acc": 0.7790993452072144, "length": 3.994722843170166, "mode": "train", "epoch": 30}
{"loss": 0.5116184949874878, "acc": 0.7587890625, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.4592222273349762, "acc": 0.7802190184593201, "length": 3.994823455810547, "mode": "train", "epoch": 31}
{"loss": 0.5167526006698608, "acc": 0.7583984732627869, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.44860026240348816, "acc": 0.7854292392730713, "length": 3.9962427616119385, "mode": "train", "epoch": 32}
{"loss": 0.504408597946167, "acc": 0.7603515982627869, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.44791266322135925, "acc": 0.7905551791191101, "length": 3.9959237575531006, "mode": "train", "epoch": 33}
{"loss": 0.5055957436561584, "acc": 0.7667968273162842, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.4414173662662506, "acc": 0.7872589230537415, "length": 3.995678186416626, "mode": "train", "epoch": 34}
{"loss": 0.5014349222183228, "acc": 0.7648438811302185, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.44289255142211914, "acc": 0.7912604808807373, "length": 3.9958136081695557, "mode": "train", "epoch": 35}
{"loss": 0.5058788061141968, "acc": 0.7617188096046448, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.43600282073020935, "acc": 0.7895109057426453, "length": 3.996847152709961, "mode": "train", "epoch": 36}
{"loss": 0.49234408140182495, "acc": 0.7740234732627869, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.43310028314590454, "acc": 0.792702853679657, "length": 3.997014045715332, "mode": "train", "epoch": 37}
{"loss": 0.5061463117599487, "acc": 0.765625, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.43150171637535095, "acc": 0.7935304045677185, "length": 3.9977071285247803, "mode": "train", "epoch": 38}
{"loss": 0.48936009407043457, "acc": 0.7767577767372131, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.42412009835243225, "acc": 0.7960113286972046, "length": 3.9978785514831543, "mode": "train", "epoch": 39}
{"loss": 0.4831528663635254, "acc": 0.7720703482627869, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.4202274978160858, "acc": 0.7973157167434692, "length": 3.9983956813812256, "mode": "train", "epoch": 40}
{"loss": 0.48642951250076294, "acc": 0.7853515148162842, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.414472371339798, "acc": 0.8004401326179504, "length": 3.99839448928833, "mode": "train", "epoch": 41}
{"loss": 0.4831625819206238, "acc": 0.779296875, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.4180314540863037, "acc": 0.7993197441101074, "length": 3.9984989166259766, "mode": "train", "epoch": 42}
{"loss": 0.4816502630710602, "acc": 0.7746093273162842, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.4130704998970032, "acc": 0.7986188530921936, "length": 3.998746633529663, "mode": "train", "epoch": 43}
{"loss": 0.4852546453475952, "acc": 0.7789062857627869, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.41176557540893555, "acc": 0.8034594655036926, "length": 3.99892258644104, "mode": "train", "epoch": 44}
{"loss": 0.47384870052337646, "acc": 0.7810547351837158, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.4071943163871765, "acc": 0.8035167455673218, "length": 3.9987525939941406, "mode": "train", "epoch": 45}
{"loss": 0.47843796014785767, "acc": 0.7798828482627869, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.40470051765441895, "acc": 0.8045299649238586, "length": 3.999274969100952, "mode": "train", "epoch": 46}
{"loss": 0.4789181351661682, "acc": 0.7826171517372131, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.4023338854312897, "acc": 0.8068899512290955, "length": 3.9992125034332275, "mode": "train", "epoch": 47}
{"loss": 0.4808058738708496, "acc": 0.7757812142372131, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.40049225091934204, "acc": 0.8065393567085266, "length": 3.9993815422058105, "mode": "train", "epoch": 48}
{"loss": 0.4871290326118469, "acc": 0.7800780534744263, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.39761456847190857, "acc": 0.8099600672721863, "length": 3.9993057250976562, "mode": "train", "epoch": 49}
{"loss": 0.47569572925567627, "acc": 0.783203125, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.3952617347240448, "acc": 0.8107330203056335, "length": 3.9995272159576416, "mode": "train", "epoch": 50}
{"loss": 0.4784606695175171, "acc": 0.7820311784744263, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.39247074723243713, "acc": 0.8132126927375793, "length": 3.999587297439575, "mode": "train", "epoch": 51}
{"loss": 0.4730526804924011, "acc": 0.7835937142372131, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.3914327621459961, "acc": 0.8169311285018921, "length": 3.9995808601379395, "mode": "train", "epoch": 52}
{"loss": 0.46943795680999756, "acc": 0.7880859375, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.3842746317386627, "acc": 0.8179921507835388, "length": 3.9995687007904053, "mode": "train", "epoch": 53}
{"loss": 0.475380003452301, "acc": 0.7843751311302185, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.3842709958553314, "acc": 0.8186411261558533, "length": 3.999492883682251, "mode": "train", "epoch": 54}
{"loss": 0.47205132246017456, "acc": 0.7874999642372131, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.37846991419792175, "acc": 0.8228911757469177, "length": 3.9995920658111572, "mode": "train", "epoch": 55}
{"loss": 0.46090441942214966, "acc": 0.7921874523162842, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.3811454176902771, "acc": 0.8234853148460388, "length": 3.9996871948242188, "mode": "train", "epoch": 56}
{"loss": 0.4658390283584595, "acc": 0.7935547232627869, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.36827710270881653, "acc": 0.8259074091911316, "length": 3.9997334480285645, "mode": "train", "epoch": 57}
{"loss": 0.4590766429901123, "acc": 0.7847656607627869, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.370734840631485, "acc": 0.8249631524085999, "length": 3.999743700027466, "mode": "train", "epoch": 58}
{"loss": 0.46090900897979736, "acc": 0.7925781607627869, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.36772865056991577, "acc": 0.8321477174758911, "length": 3.9993557929992676, "mode": "train", "epoch": 59}
{"loss": 0.4648664593696594, "acc": 0.7953124642372131, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.36733123660087585, "acc": 0.832636296749115, "length": 3.99975323677063, "mode": "train", "epoch": 60}
{"loss": 0.45093780755996704, "acc": 0.7962890267372131, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7100000977516174
